### PR TITLE
page home

### PR DESCRIPTION
**Pull request by marioprasetyo**

### CODE REVIEW

**Comment by triandamai** [comment]
convoy
```dart

new NestedScrollView(
        body: new Column(
          children: <Widget>[
            new Container(
              height: 300.0,
              child: new ListView.builder(
                itemCount: 60,//jumlah data 
                itemBuilder: (BuildContext context, int index) {
 
                  return ItemContinueListening();
                },
              ),
            ),
            new Container(
              height: 300.0,
              child: new ListView.builder(
                itemCount: 60,
                itemBuilder: (BuildContext context, int index) {
                  return ItemEpisode();
                },
              ),
            ),

          ],
        ),
      )
```

**Review by triandamai — CHANGES_REQUESTED** [review]
repisi

**Comment by marioprasetyo** [comment]
belom paham mas

File: lib/components/card_audiobook.dart [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    required this.index,
+    required this.totalIndex,
+  });
+
+  final int totalIndex;
+  final String image;
+  final String name;
+  final String description;
+  final int index;
+
```
Comment (triandamai) [inline lib/components/card_audiobook.dart:21]: onClick

File: lib/components/card_continuelistening.dart [hunk 2]
```diff
@@ -9,8 +9,11 @@ class ContinueListening extends StatefulWidget {
       {super.key,
       required this.image,
       required this.name,
-      required this.percent});
+      required this.percent,
+      required this.index, required this.totalIndex});
```
Comment (triandamai) [inline lib/components/card_continuelistening.dart:14]: onClick

File: lib/components/card_episode.dart [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
+      {super.key,
+      required this.name,
+      required this.artist,
+      required this.image,
+      required this.time,
+      required this.index,
+      required this.totalIndex});
+  final int totalIndex;
+  final int index;
   final String name;
```
Comment (triandamai) [inline lib/components/card_episode.dart:18]: onClick

**Pull request merged**